visualizer/service/api/__init__.py

```diff
@@ -60,9 +70,34 @@
         if 'influxdb' == datasource:
             influxdb_datasource_name = config.get("influxdb", "name")
             influxdb_datasource_type = config.get("influxdb", "type")
-            influxdb_datasource_url = config.get("influxdb", "url")
             influxdb_datasource_access = config.get("influxdb", "access")
             
 except Exception as e:
     print "Error: %s" % e.message
     quit()
+
+""" Gets the IP address of one a the node contained
+    in a Kubernetes cluster
+
+    Raises:
+        Exception -- It was not possible to connect with the
+        Kubernetes cluster.
+
+    Returns:
+        string -- The node IP
+    """
```

Comment:
nit: indentation issues and an unnecessary blank line between docstring and method it describes.

Reply:
Modification made in commit 5cddb79d05f7fb24c82ab6018736cd0ab7adfd40